bin/crew

```diff
@@ -1158,8 +1177,10 @@ def install
     # perform install process
     install_package dest_dir
 
-    # perform post-install process
-    post_install dest_dir
+    unless @resolve_dependencies_and_install == 1 or @resolve_dependencies_and_build == 1
```

Comment:
I'm confused why we would even need this conditional.

Reply:
Otherwise, postinstall is run twice for the first package we are trying to install.

Reply:
This whole process seems overly complicated to me.

Reply:
It's an artifact of the current decision tree.

Both `resolve_dependencies_and_install` & `resolve_dependencies_and_build` will invoke `post_install` on a notional `package 1` after finding all dependencies, since it creates an array of all _dependencies for_ `package 1`, and also adds `package 1` to that array, which it can then use to run `post_install`s for everything in that array in sequence.  So we need to exclude `package 1` here, otherwise, `package 1` gets `post_install` run twice.

Reply:
Would it make sense to remove the entire code block since post_install is called in another place?

Reply:
No, because you need if there if you are just calling install on a package w/o dependencies which need installing.

Just trying to take a light touch here with a minimum of refactoring.

Reply:
It just doesn't feel elegant but I guess it works for now.

Reply:
Did a clean install with buildessential in a container and found no issues.